src/style.css

```diff
@@ -0,0 +1,60 @@
+body {
+  margin: 0;
+  background-color: #eee;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  height: 60px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  position: relative;
+  top: 10px;
+}
+
+.nav {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.link {
+  padding: 22.5px 0;
```

Comment:
try to avoid float numbers, you can use line-height equal height of your container in this case